subcommands/keys/ca_create.go

```diff
@@ -84,12 +85,13 @@ func doCreateCA(cmd *cobra.Command, args []string) {
 
 	if len(hsmModule) > 0 {
 		if len(hsmPin) == 0 {
-			fmt.Println("ERROR: --hsm-pin is required with --hsm-module")
-			os.Exit(1)
+			subcommands.DieNotNil(errors.New("--hsm-pin is required with --hsm-module"))
```

Comment:
No need to check `hsmTokenLabel` value similarly?

Reply:
We have a default value for the label... which I don't really like, but still.

That said, I actually like the idea to require this argument, so that we kind of remind the user once again that the HSM token should be configured properly.
This change would not be breaking, as the HSM support was anyway kind of broken in past versions.

Good catch! I will add a separate commit for that.

Reply:
https://github.com/foundriesio/fioctl/pull/278/commits/bfe925df2ad3748c975d5d86a292a8a4da8e977e